downstairs/src/lib.rs

```diff
@@ -2057,11 +2057,10 @@ async fn recv_task<RT>(
 ) where
     RT: tokio::io::AsyncRead + std::marker::Unpin + std::marker::Send,
 {
-    // How long we wait before logging a message that we have not heard from
-    // the upstairs.
-    const TIMEOUT_SECS: f32 = 15.0;
-    // How many timeouts will tolerate before we disconnect from the upstairs.
-    const TIMEOUT_LIMIT: usize = 3;
+    const TIMEOUT: VerboseTimeout = VerboseTimeout {
```

Comment:
I think it's worth keeping the comments here for what these values represent.

Though (reading further down), maybe better to put at `CLIENT_TIMEOUT`

Reply:
Added comments in 6cc9128